src/bp-notifications/bp-notifications-settings.php

```diff
@@ -451,11 +458,35 @@ function bb_admin_setting_callback_notification_warning() {
  * @param bool  $checked Is checked or not.
  */
 function bb_activate_notification( $field, $checked ) {
-	$label = ( ! empty( $field['admin_label'] ) ? $field['admin_label'] : $field['label'] );
+	$label        = ( ! empty( $field['admin_label'] ) ? $field['admin_label'] : $field['label'] );
+	$tooltip_pos  = '';
+	$tooltip_text = '';
+	if ( ! empty( $field['notification_tooltip_text'] ) ) {
+		$tooltip_pos  = 'up';
+		$tooltip_text = $field['notification_tooltip_text'];
+	}
+	$disabled = ! empty( $field['notification_read_only'] ) && true === $field['notification_read_only'];
 	?>
 
 	<input name="bb_enabled_notification[<?php echo esc_attr( $field['key'] ); ?>][main]" type="hidden" value="no" />
-	<input class="bb-notification-checkbox" id="bb_enabled_notification_<?php echo esc_attr( $field['key'] ); ?>" name="bb_enabled_notification[<?php echo esc_attr( $field['key'] ); ?>][main]" type="checkbox" value="yes" <?php checked( $checked, 1 ); ?> />
+	<span class="forum-subscription-input"
```

Comment:
Change the class name to target globally for all, not for the subscription specific.